train_configs/debug_model.toml

```diff
@@ -30,6 +30,7 @@ name = "AdamW"
 lr = 8e-4
 
 [training]
+seed = 1  # use 1 avoid determinism
```

Comment:
Since the default value is `None` we can remove this flag in toml file.

Reply:
removed